app/models/calendar_event.rb

```diff
@@ -0,0 +1,125 @@
+class CalendarEvent
+  include ActiveModel::Serialization
+
+  attr_accessor :start_date
+  attr_accessor :title
+  attr_accessor :html  # transient value populated by controller and serialized as 'title'
+
+  attr_accessor :start
+  attr_accessor :title
+  attr_accessor :backgroundColor
+
+  attr_accessor :event_type
+  attr_accessor :num_of_logs
+
+  # JE Todo: Confirm if potentially useful and worth including
+  attr_accessor :model_type  #Loan/ProjectStep
+  attr_accessor :model_id
```

Comment:
If we need it we'll add it later. As you may have noticed, this is my general philosophy.